tests/test_filtering.py

```diff
@@ -62,6 +64,13 @@ class WidgetListView(GenericModelView):
 
         filtering = Filtering(
             color=operator.eq,
```

Comment:
nothing prevents you from doing so right now. and i'm pretty sure we do share filter objects, e.g. from decorated functions